editor/src/components/canvas/controls/select-mode/cursor-overlay.tsx

```diff
@@ -53,3 +53,21 @@ export const CursorOverlay = React.memo(() => {
     portalDiv,
   )
 })
+
+export interface WorldState {
```

Comment:
Ha! Please rename this 😁 Or tbh I'm not even sure it's needed? We could just provide the two params separately to the function below?